_includes/tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/harness-console-consumer-keys.sh

```diff
@@ -0,0 +1 @@
+ccloud kafka topic consume mytopic --print-key --from-beginning 
```

Comment:
FYC - remove  `--from-beginning` flag, it's a lot of data on the terminal

Reply:
Ack, resolved in https://github.com/confluentinc/kafka-tutorials/pull/496/commits/5d3dd56cb80808f3662e3794a54cd93feb7fc212